apps/finance/app/src/components/Balances.js

```diff
@@ -18,9 +18,10 @@ function useBalanceItems(balances) {
         return {
           address,
           amount,
-          convertedAmount: convertRates[symbol]
-            ? getConvertedAmount(amount, convertRates[symbol], decimals)
-            : new BN('-1'),
+          amountConverted:
+            amount && decimals && convertRates[symbol]
+              ? convertAmount(amount, decimals, 1 / convertRates[symbol])
```

Comment:
Note that we can also get a `convertRate` that is > 1 here (potentially, many times greater).